fbximport/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Build frontend
+FROM ubuntu:noble as build
+RUN apt update -y; \
+    apt upgrade -y; \
+    apt install -y build-essential catch2 cmake wget libxml2-dev; \
+    apt clean -y
+WORKDIR /fbximporter
+COPY . .
+RUN wget --no-check-certificate -O fbxsdk.tar.gz https://damassets.autodesk.net/content/dam/autodesk/www/files/fbx202037_fbxsdk_gcc_linux.tar.gz
```

Comment:
Consider linkign to where you found this url, and where to find an updated one in the future